### ISSUE TITLE
Filters - hanging indents

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-policies-prototype#185]
<p>instead of:<br>
  [_] None (12 items)<br>
  [_] WCAG 1.0<br>
  derivative (2 items)<br>
  [_] WCAG 2.0<br>
  (22 items)<br>
  [_] WCAG 2.0<br>
  derivative (4 items)</p>
<p>should be:<br>
  [_] None (12 items)<br>
  [_] WCAG 1.0<br>
  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;derivative (2 items)<br>
  [_] WCAG 2.0<br>
  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(22 items)<br>
  [_] WCAG 2.0<br>
  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;derivative (4 items)<br>
</p>

I guess I could live with this for 2 weeks if need be.
